### PR TITLE
Put aktualizr-cert-provider in garage_deploy.deb

### PR DESCRIPTION
**Pull request by lbonn**
For convenience

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -334,7 +334,7 @@ add_library(jsoncpp OBJECT third_party/jsoncpp/jsoncpp.cpp)
 
 # General packaging configuration
 set(CPACK_GENERATOR "DEB")
-set(CPACK_DEBIAN_PACKAGE_MAINTAINER "anton.gerasimov@here.com")
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER "patrick.vacek@here.com")
```
Comment (patrickvacek) [inline CMakeLists.txt]: lol

**Comment by zabbal** [comment]
Seems reasonable at a quick glance, provided you can fix the CI.

File: src/libaktualizr/package_manager/CMakeLists.txt [hunk 2]
```diff
@@ -8,9 +8,11 @@ set(HEADERS packagemanagerconfig.h
             packagemanagerfake.h
             packagemanagerinterface.h)
 
-add_library(package_manager OBJECT ${SOURCES})
+add_library(package_manager OBJECT packagemanagerfactory.cc packagemanagerfake.cc packagemanagerinterface.cc)
```
Comment (patrickvacek) [inline src/libaktualizr/package_manager/CMakeLists.txt]: This is a bit less than ideal; should we instead remove `packagemanagerconfig.cc` from `SOURCES`?

File: CMakeLists.txt (hunk 1 above)
Comment (lbonn) [reply CMakeLists.txt]: maybe I should have asked you before indeed, sorry

**Comment by lbonn** [comment]
Hm yes I really don't know what's going on with CI, that's quite annoying.

File: CMakeLists.txt (hunk 1 above)
Comment (patrickvacek) [reply CMakeLists.txt]: It's fine! I was just amused.

File: src/libaktualizr/package_manager/CMakeLists.txt (hunk 2 above)
Comment (lbonn) [reply src/libaktualizr/package_manager/CMakeLists.txt]: ok changed

**Pull request merged**